src/main/java/com/manufacturing/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,109 @@
+package com.manufacturing.dao.jdbc;
+
+import com.manufacturing.dao.ManufacturerDao;
+import com.manufacturing.lib.Dao;
+import com.manufacturing.model.Manufacturer;
+import com.manufacturing.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    private static final String MANUFACTURER_NAME = "manufacturer_name";
+    private static final String MANUFACTURER_ID = "manufacturer_id";
+    private static final String ORIGIN = "origin";
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers (manufacturer_name, origin) VALUE (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query,
```

Comment:
Weird indentation + this statement should be closed explicitly

Reply:
done, but can you explain why we can't close it in try-with-resources??

Reply:
Statements and ResultSets of the specific Connection instance should be closed when we call .close() on the Connection, but it may not happen. You can use try with resources for PreparedStatement declaration, but it's redundant, common .close() call is enough